README.md

```diff
@@ -2,4 +2,159 @@
+
+5. Set up two class methods called `validSuits` and `validRanks` that return arrays containing the four suit icons and the thirteen string representations of the ranks (Ace through King).
+
+### `Deck.swift`
+
+1. Re-enable the target membership for `DeckSpec.swift` for `swift-blackjackTests`.
+
+2. Create a class file for the `Deck` class. It should have two private array properties for holding cards: one for holding the remaining (undealt) cards, and the other for holding the dealt cards. The test file cannot see them so name them yourself.
+
+3. Write an initializer that generates the fifty-two unique cards to represent a standard deck of playing cards and holds them in the array of cards that can be dealt.
+
+4. Make the `Deck` class conform to the `CustomDebugStringConvertible` protocol. The deck should print the count of its remaining cards and dealt cards to the console. Look at the tests to see the exact substrings they are expecting.  
+**Top-tip:** *Set this up as a calculated property.*
+
+5. Write the `drawCard()` function to return the next card. It should remove that method from the remaining cards and add it to the dealt cards.
+
+6. Write the `reshuffle()` method to randomize the cards that have not yet been dealt out.  
+**Hint:** *The* `arc4random_uniform()` *C function is accessible in Swift also.*
+
+7. Write the `shuffle()` method that shuffles the dealt cards back into the cards that are ready to deal, randomizing them again.
+
+### `Player.swift`
+
+1. Re-enable the target membership for `PlayerSpec.swift` for `swift-blackjackTests`.
+
+2. Create a new class file for the `Player` class. Have it conform to the `CustomDebugStringConvertible` protocol. It will need nine properties total:
+  * `name`, a string,
+  * `cards`, an array of `Card` objects,
+  * `handscore`, an unsigned integer,
+  * `blackjack`, a boolean,
+  * `busted`, a boolean,
+  * `stayed`, a boolean,
+  * `mayHit`, a boolean,
+  * `wallet`, an unsigned integer, and
+  * `debugDescription`, a string.
+
+3. Write an initializer which takes an argument for the `name` property. Make the initial value of the `wallet` property `100`.
+
+4. Make `debugDescription` a calculated property which returns a string detailing the object's current state.
+
+5. Make `handscore` a calculated property that evaluates the `cards` array. It may use one Ace as an "eleven" (11) if doing so will not bust the hand.
+
+6. Make `blackjack` a calculated property that determines whether the hand is a blackjack (a score of 21 with only two cards).
+
+7. Make `busted` a calculated property that determines whether the hand is busted (over score of 21).
+
+8. Leave the `stayed` property initialized to `false`. It will need to be used to hold state during a round.
+
+9. Make `mayHit` a calculated property that determines whether the player may take a new card (if the hand is not busted, is not a blackjack, and if the player has not stayed).
```

Comment:
also why is this a property? This is not some sort of state right? Effectively you are asking the object a question which sounds more like a behavior